browsers/node14.15.0-chrome85-ff81/Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM cypress/base:14.15.0
+
+USER root
+
+RUN node --version
+
+# Chrome dependencies
+RUN apt-get update
+RUN apt-get install -y fonts-liberation libappindicator3-1 xdg-utils
+
+# install Chrome browser
+ENV CHROME_VERSION 85.0.4183.121
```

Comment:
fwiw I tried to get 86 (https://chromereleases.googleblog.com/2020/10/stable-channel-update-for-desktop_20.html) but the deb file gave 404. I didn't attempt to dig into it